test/test_pipeline/components/training/test_training.py

```diff
@@ -165,7 +165,8 @@ def test_regression_epoch_training(self):
          _,
          epochs,
          logger) = self.prepare_trainer(StandardTrainer(),
-                                        constants.TABULAR_REGRESSION)
+                                        constants.TABULAR_REGRESSION,
+                                        1000)
```

Comment:
Can we rewrite like this? (Same for the rest)
```
EPOCH = 1000

... = self.prepare_trainer(StandardTrainer(),
                                       constants.TABULAR_REGRESSION)
                                       constants.TABULAR_REGRESSION,
                                       epoch=1000)

```

Reply:
I can do this sure. but I didn't do it originally because its only used in two of the tests

Reply:
Ok, that also makes sense, but I just found out last time when I fixed the variable values in `conftest.py`, it was just annoying.